python/proton/_data.py

```diff
@@ -1158,7 +1154,7 @@ def get_map(self) -> int:
         """
         return pn_data_get_map(self._data)
 
-    def get_array(self):
+    def get_array(self) -> Tuple[int, bool, Optional[int]]:
```

Comment:
```suggestion
    def get_array(self) -> Optional[Tuple[int, bool, int]]:
```

hmm, possibly the correct type is actually this; in any case, what the doc used to say was wrong

Reply:
actually, ^^^ cannot be right